src/v/cloud_storage/remote_partition.cc

```diff
@@ -496,12 +496,14 @@ kafka::offset remote_partition::first_uploaded_offset() {
     return so;
 }
 
-model::offset remote_partition::last_uploaded_offset() {
+kafka::offset remote_partition::last_uploaded_offset() {
     vassert(
       _manifest.size() > 0,
       "The manifest for {} is not expected to be empty",
       _manifest.get_ntp());
-    return _manifest.get_last_offset();
+    auto last = _manifest.get_last_kafka_offset().value();
+    vlog(_ctxlog.trace, "remote partition last_kafka_offset: {}", last);
```

Comment:
nit: use `debug` level, most of `cloud_storage` and `archival` is using `debug` for the most part. The `trace` is used to log all `http` requests and a lot of `s3` related messages which are very spammy.

Reply:
Sure, I'll follow suit with the trace log above